graph/articles/nullable.md

```diff
@@ -0,0 +1,311 @@
+# Nullable Properties
```

Comment:
We should use consistent section numbering schemes.  This article uses something different from the other 2 articles.